server/data/supabaseTypes.ts

```diff
@@ -34,6 +34,78 @@ export type Database = {
   }
   public: {
     Tables: {
+      helper_parties: {
+        Row: {
+          created_at: string
+          display_name: string
+          uuid: string
+        }
+        Insert: {
+          created_at?: string
+          display_name: string
+          uuid?: string
+        }
+        Update: {
+          created_at?: string
+          display_name?: string
+          uuid?: string
+        }
+        Relationships: []
+      }
+      helper_party_network_members: {
+        Row: {
+          created_at: string
+          helper_party_network_uuid: string
+          helper_party_uuid: string
+        }
+        Insert: {
+          created_at?: string
+          helper_party_network_uuid: string
+          helper_party_uuid: string
+        }
+        Update: {
+          created_at?: string
```

Comment:
what does this mean for an update operation if someone specifies `created_at`?

Reply:
this is an autogenerated file, generating types for the tables. I don't believe there is a way at the DB level to assure that `created_at` isn't modified.

Reply:
yea unless this generates a JS API for DB, it is not possible to prevent that